apps/actions-dependencies-updater/src/git-cli.mts

```diff
@@ -0,0 +1,69 @@
+import * as log from "./logger.mjs";
```

Comment:
I found https://www.npmjs.com/package/simple-git to be pretty nice when I was working with signed changesets

Reply:
What I have works well. I found there's problems with libraries not respecting the env vars needed for commit signing.